_includes/search-results.html

```diff
@@ -1,3 +1,29 @@
 <div class="search__results">
-  <p class="search__results-count">You haven't searched for anything yet.</p>
+</div>
+<div class="search__algolia">
+  <p>Search powered by
+    <a href="https://www.algolia.com/">
+      <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Let's try to put the SVG in an include so we can run [gulp-svgmin](https://github.com/ben-eb/gulp-svgmin) against it.